index.html

```diff
@@ -329,8 +345,30 @@
       cmds.join('<br/>&nbsp;&nbsp;/');
     box.scrollTop = box.scrollHeight - 10;
   },
+  rosterize: function() {
+    var all = box.innerHTML.split("<br>").
+              map(function(s) { return !!s ? s.split(":")[0] : ""; });
+    all.forEach(addSpeaker);
+  },
 };
 
+// "Roster" management
+if (localStorage.roster && 0 < localStorage.roster.length) {
+  roster = JSON.parse(localStorage.roster);
+}
+function addSpeaker(nick) {
+  if (!nick) { return; }
+  
+  roster[nick] = nick;
```

Comment:
there is no builtin Array.prototype.unique(), and this is cheaper than writing it.